src/App.jsx

```diff
@@ -0,0 +1,76 @@
+import { Routes, Route, Navigate } from 'react-router-dom';
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import Navbar from './components/Navbar';
+
+import AuthLayout from './layouts/AuthLayout';
+import ProtectedRoute from './layouts/ProtectedRoute';
+
+import Login from './routes/Login';
+import Signup from './routes/Signup';
+import Courses from './routes/Courses';
+import Details from './routes/Details';
+import Reserve from './routes/Reserve';
+import Reservations from './routes/Reservations';
+import AddCourse from './routes/AddCourse';
+import DeleteCourse from './routes/DeleteCourse';
+
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 3):

![image](https://github.com/smunoz1988/vecs-school-frontend/assets/57421823/c4050c16-b31f-4e9c-b1ff-c59155637c08)


That way, your code will meet the latest standards. 💪

Reply:
@Meltrust Thanks for the suggestion! 

We refactor our code and all functions are arrow functions.